### PR TITLE
Add Webhook Health Endpoint

### PR DESCRIPTION
**Pull request by dlitvakb**

### CODE REVIEW

File: spec/lib/contentful/management/webhook_health_spec.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+      describe '.all' do
+        it 'is not supported' do
+          expect { subject.all }.to raise_error 'Not supported'
+        end
+      end
+
+      describe '.find' do
+        it 'class method also works' do
+          vcr('webhook_health/find') { expect(Contentful::Management::WebhookHealth.find(client, space_id, webhook_id)).to be_kind_of Contentful::Management::WebhookHealth }
```
Comment (mariobodemann) [inline spec/lib/contentful/management/webhook_health_spec.rb]: That is a super long line (understanding wise and character wise) Can you make sure to check also the content and not just the type of the response of `find`?

Comment (dlitvakb) [reply spec/lib/contentful/management/webhook_health_spec.rb]: The contents are tested on the last test, this is a call to a wrapper that ends up calling the interface in the last 2 tests. So I just test that the correct kind of thing is obtained, as the values are already tested.

File: lib/contentful/management/client.rb [hunk 2]
```diff
@@ -200,6 +201,15 @@ def webhook_calls
         ClientWebhookCallMethodsFactory.new(self)
       end
 
+      # Allows manipulation of webhook health in context of the current client
+      # Allows listing all webhook call details for client and finding one by ID.
```
Comment (dborsatto) [inline lib/contentful/management/client.rb]: Is this correct, or it is copy/paste gone wrong?

Comment (dlitvakb) [reply lib/contentful/management/client.rb]: copy paste error, will fix

Comment (dlitvakb) [reply lib/contentful/management/client.rb]: fixed

**Pull request merged**